common/scripts/ci/docker_rushx.sh

```diff
@@ -3,7 +3,7 @@
 # Absolute root directory - for volumes
 ROOT_DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}")/../../.. && pwd -P))
 
```

Comment:
is there a special reason for this?, if not please keep it as it is for this scope of change